### ISSUE TITLE
The loss is NAN

### ISSUE THREAD

**Issue by danieljf24** [mosessoh/CNN-LSTM-Caption-Generator#5]
Hi, author. I trained the model on the ubuntu 14.04 with CUDA 7.5, cudnn 0.4 and tensorflow 0.10, but the returned loss is always NAN. I don't know how to fix it.

**Comment by danieljf24** [comment]
I change the tensorflow from 0.10 to 0.08, and it works.

But I may find a bug in the line 44 of model.py
`b_i = tf.get_variable('b_i', shape=[self.config.batch_size, self.config.embed_dim])` .   

Is the following code better?
'b_i = tf.get_variable('b_i', shape=[self.config.embed_dim])'

**Comment by 201528014227051** [comment]
Hi, I try to download use the download.sh， but it's pity that the source is not available now, could u please give me the downloaded data if u still keep it? my email is binqiang2wang@qq.com. thanks anyway!

**Comment by mosessoh** [comment]
Please see my response in #12 . I've created a new download.sh script that should fix the issues you're facing. You should now be able to use the new download.sh script by cloning/pulling the repo. If you're still interested in the project, do let me know if you're facing any issues and I will be happy to try and help.
